app/controllers/application_controller.rb

```diff
@@ -1,6 +1,17 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
+  before_action :update_allowed_parameters, if: :devise_controller?
+
+  protected
+
+  def update_allowed_parameters
+    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :bio, :photo, :email, :password) }
+    devise_parameter_sanitizer.permit(:account_update) do |u|
+      u.permit(:name, :bio, :photo, :email, :password, :current_password)
+    end
+  end
+
   helper_method :current_user
 
   def current_user
```

Comment:
- [ ] as part of the requirements kindly remove the current_user method since devise will manage it from now on 🙏